uptane-standard.md

```diff
@@ -376,18 +376,18 @@ A repository contains images and metadata. Each role has a particular type of me
 
 ### The Root role {#root_role}
 
-The Root role SHALL be responsible for a Certificate Authority as defined in {{RFC3647}}.
-The Root role SHALL produce and sign Root metadata as described in {{root_meta}}.
-The Root role SHALL sign the public keys used to verify the metadata produced by the Timestamp, Snapshot, and Targets roles.
-The Root role SHALL revoke keys for the other roles, in case of compromise.
+A repository's Root role SHALL be responsible for a Certificate Authority as defined in {{RFC3647}}.
+A repository's Root role SHALL produce and sign Root metadata as described in {{root_meta}}.
+A repository's Root role SHALL sign the public keys used to verify the metadata produced by the Timestamp, Snapshot, and Targets roles.
+A repository's Root role SHALL revoke keys for the other roles, in case of compromise.
 
 ### The Targets role {#targets_role}
 
-The Targets role SHALL produce and sign metadata about images and delegations as described in {{targets_meta}}.
+A repository's Targets role SHALL produce and sign metadata about images and delegations as described in {{targets_meta}}.
 
 #### Delegations {#targets_role_delegations}
 
-The Targets role on the Image repository MAY delegate the responsibility of signing metadata to other, custom-defined roles. If it does, it MUST do so as specified in {{delegations_meta}}.
+The top-level Targets role on the Image repository MAY delegate the responsibility of signing metadata to other, custom-defined roles referred to as delegated targets. If it does, it MUST do so as specified in {{delegations_meta}}.
```

Comment:
Why "top-level" only here? Sub-delegations are permitted.

Reply:
Good catch, I updated it to allow for sub-delegations.